web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -888,6 +887,16 @@ $.extend(fixmystreet.set_up, {
 
 });
 
+fixmystreet.update_councils_text = function(data) {
+    $('#js-councils_text').html(data.councils_text);
+    $('#js-councils_text_private').html(data.councils_text_private);
+
+    var single_body_only = $('#single_body_only').val();
+    if (single_body_only) {
+        $('#js-councils_text strong').html(single_body_only);
```

Comment:
Hmm, I think this should work? Here it is for me:
![bromleytfl](https://user-images.githubusercontent.com/154364/41343008-739190ee-6ef5-11e8-8687-17b9f8e859cf.gif)

It will do what you say if you pick a category that is in both the Bromley and TfL bodies, but that won't be the case, they'll be in either one or the other.

Reply:
Ah, my dev server has a "Road defect" category for both Bromley and TfL which is why this is happening I think.

Reply:
Yeah, I'll try and do it more generically than assuming no overlap.